userspace/libsinsp/logger.cpp

```diff
@@ -205,3 +271,84 @@ char* sinsp_logger::format(const char* fmt, ...)
 
 	return m_tbuf;
 }
+
+const char *sinsp_logger::encode_severity(const severity sev)
+{
+	// All severity strings should be ENCODE_LEN chars long
+	static const std::string pre("SEV");
```

Comment:
These should be defined along side the enum where we define the severities, and probably can be put in an array we can directly index.